mit_frontend/src/routers/UserInfo.js

```diff
@@ -109,23 +117,28 @@ class UserInfo extends Component {
              */
             <p>Name</p>
             <TextField
-              errorText="This field is required"
+              // required
               hintText="First Name"
               floatingLabelText="First Name"
               floatingLabelFocusStyle={styles.textField.text}
               underlineFocusStyle={styles.textField.underline}
               value={this.state.firstName}
+              minLength="2"
+              maxLength="15"
               onChange={
-                (event) => {this.setState({firstName: event.target.value})}
+                (event) => {this.setState({lastName: event.target.value})}
```

Comment:
Should setState firstName rather than lastName. This gonna change the value in lastName field.